packages/terra-consumer-layout/tests/nightwatch/DexLayout.jsx

```diff
@@ -129,6 +129,17 @@ const data = {
       target: '_blank',
     },
   ],
+  // alertBanner
+  siteAlert: [
+    {
+      alertId: '001',
+      alertMessage: 'Alert Banner text',
+      handleDismiss: () => {} },
+    {
+      alertId: 'null',
+      alertMessage: <a href="">Link text </a>,
+      handleDismiss: () => {} },
```

Comment:
can you make these dismiss buttons work for testing?
https://terra-consumer-pr-90.herokuapp.com/static/#/tests/layout-tests/dex

Reply:
Ok. I will create a dummy function to test it.